src/data/web3.js

```diff
@@ -264,7 +263,7 @@ export const useEthersProviderContext = singletonHook(
 
         const previouslyConnectedWallets = ls.get(web3onboardKey) || [];
 
-        if (previouslyConnectedWallets) {
+        if (previouslyConnectedWallets.length) {
```

Comment:
this was giving an error locally when there were no `previouslyConnectedWallets`